install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -295,6 +295,9 @@ data:
   enable-xt-socket-fallback: {{ .Values.global.enableXTSocketFallback | quote }}
   install-iptables-rules: {{ .Values.global.installIptablesRules | quote }}
   auto-direct-node-routes: {{ .Values.global.autoDirectNodeRoutes | quote }}
+  {{- if .Values.global.nativeRoutingCIDR }}
```

Comment:
Small nit, can you set the same indentation of this if statement the same it is set for the ones in the previous lines?

Reply:
Done.